econet_hvac_furnace.yaml

```diff
@@ -24,3 +73,112 @@ sensor:
     unit_of_measurement: "°F"
     state_class: "measurement"
     entity_category: "diagnostic"
+    filters:
+      - delta: 0.9
+  - platform: template
+    name: "Flame Sensor"
+    id: flame_sensor
+    entity_category: "diagnostic"
+    accuracy_decimals: 1
+    unit_of_measurement: "uA"
+    filters:
+      - delta: 0.09
+  - platform: econet
+    name: "Furnace AFUE"
+    id: furnace_afue
+    sensor_datapoint: FURNAFUE
+    entity_category: "diagnostic"
+    request_once: true
+  - platform: econet
+    name: "Furnace High Heat BTU Output"
+    id: furnace_hh_btus
+    sensor_datapoint: FURNHBTU
+    entity_category: "diagnostic"
+    request_once: true
+  - platform: econet
+    name: "Furnace Low Heat BTU Output"
+    id: furnace_lh_btus
+    sensor_datapoint: FURNLBTU
+    entity_category: "diagnostic"
+    request_once: true
+  - platform: template
+    name: "Lifetime Hours - Low Heat"
+    id: lh_lh
+    entity_category: "diagnostic"
+    accuracy_decimals: 0
+    filters:
+      - delta: 0.9
+  - platform: template
+    name: "Lifetime Hours - High Heat"
+    id: hh_lh
+    entity_category: "diagnostic"
+    accuracy_decimals: 0
+    filters:
+      - delta: 0.9
+  - platform: template
+    name: "Instant BTU Output"
+    id: instant_btu_output
+    unit_of_measurement: "kbtu/h"
+    accuracy_decimals: 3
+    filters:
+      - delta: 0.9
+  - platform: template
+    name: "Instant BTU Usage"
+    id: instant_btu_usage
+    unit_of_measurement: "kbtu/h"
+    accuracy_decimals: 3
+    filters:
+      - delta: 0.9
+  - platform: template
+    name: "Instant Gas Usage"
+    id: gas_usage_m3_h
+    unit_of_measurement: "m³/h"
+    accuracy_decimals: 3
```

Comment:
Please include a device_class and/or icon_class for every entity so that they get a reasonable treatment in the Home Assistant frontend. I think ideally you shoudld also include a state_class for each of these entities as well.